air-dsl/Cargo.toml

```diff
@@ -10,3 +10,6 @@ codegen-winter = { package = "codegen-winter", path = "../codegen-winter", versi
 structopt = "0.3.26"
 env_logger = "0.9"
 log = { version = "0.4", default-features = false }
+winter-math = { package = "winter-math", version = "0.4", default-features = false }
+winter-utils = { package = "winter-utils", version = "0.4", default-features = false }
+winter-air = { package = "winter-air", version = "0.4", default-features = false }
```

Comment:
nit: these imports don't seem to be required by the changes in this PR. I'm guessing maybe they're needed for the testing PR instead?

Reply:
I was using them to test for compilation issues. Removed.